composer.json

```diff
@@ -30,5 +30,9 @@
         "pear-pear/HTTP_Request2": "*",
         "simple-cas/simple-cas": "dev-master",
         "zaininnari/html-minifier": "*"
+    },
+    "config": {
+      "secure-http": false
```

Comment:
This is just for pear? Why couldn't you change the repo above to use https? pear.php.net _should_ support https?